themes/src/main/resources/theme/keycloak.v2/account/src/app/content/my-resources-page/ShareTheResource.tsx

```diff
@@ -208,14 +208,12 @@ export class ShareTheResource extends React.Component<ShareTheResourceProps, Sha
                                         </GalleryItem>
 
                                 </Gallery>
-                                <ChipGroup withToolbar>
-                                    <ChipGroupToolbarItem key='users-selected' categoryName='Share with '>
+                                <ChipGroup categoryName='Share with '>
```

Comment:
Text should be translated.

Reply:
Resolved, along with a few other aria labels on this page.